autocomplete/views.py

```diff
@@ -0,0 +1,92 @@
+from dal.views import ViewMixin
```

Comment:
Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/autocomplete/views.py
+++ b/autocomplete/views.py
@@ -1,10 +1,11 @@
+import json
+
+import requests
+
 from dal.views import ViewMixin
 from dal_select2.views import Select2ViewMixin
+from django.core.exceptions import SuspiciousOperation
 from django.views.generic import View
-from django.core.exceptions import SuspiciousOperation
-import requests
-import json
-
 
 HAL_API_AUTHOR_STRUCTURE = (
     'https://api.archives-ouvertes.fr/search/authorstructure/'
```